numpy/core/memmap.py

```diff
@@ -104,7 +104,7 @@ class memmap(ndarray):
 
     See also
     --------
-    lib.format.open_memmap : Create or load a memory-mapped ``.npy`` file.
+    `lib.format.open_memmap <https://github.com/numpy/numpy/blob/v1.16.1/numpy/lib/format.py>`_ : Create or load a memory-mapped ``.npy`` file.
```

Comment:
This link isn't the best, there should be a better way to do this and also keep the line at reasonable length. This is also not the only reference that needs fixing @mattip Suggestions? `format.py` seems to lack an `__all__`.

Reply:
The link should be created automatically once the we link in the documentation in `lib.format`. Adding a link to github is wrong.

Reply:
@charris and @mattip Thanks for the review. I will try to fix it today.